tests/security/selinux/sestatus.pm

```diff
@@ -35,7 +36,11 @@ sub run {
     validate_script_output("sestatus", sub { m/SELinux status: .*disabled/ });
 
     # workaround for "selinux-auto-relabel" in case: auto relabel then trigger reboot
-    assert_script_run("sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=8/\' /etc/default/grub");
+    # NOTE: on aarch64 if reboot wait_boot() will wait grub so get rid of aarch64
+    my $results = script_run("zypper --non-interactive se selinux-auto-relabel");
```

Comment:
Cool.... 
I think we you can remote the line started from #NOTE, LGTM

Reply:
just wondering the package **selinux-auto-relabel** is only for aarch64 ?

Reply:
> Cool....
> I think we you can remote the line started from #NOTE, LGTM

Done.